test/test_fourier.jl

```diff
@@ -0,0 +1,419 @@
+function compare_tables(table1, table2; rtol=0.001, discard = [])
+
+    s_discard = Symbol.(discard)
+    test_result = true
+
+    for key in propertynames(table1)
+        if key in s_discard
+            continue
+        end
+        oe, oc = getproperty(table1,key), getproperty(table1,key)
+        if oe isa Integer || oe isa String
+            if !(oe==oc) 
+                test_result = false
+                break
+            end
+        elseif isnothing(oe)
+            if !(isnothing(oc)) 
+                test_result = false
+                break
+            end
+        else
+            if !(≈(oe,oc,rtol=rtol)) 
+                test_result = false
+                break
+            end
+        end
+    end
+
+    table1 = Metadata.drop_metadata(table1)
+    table2 = Metadata.drop_metadata(table2)
+
+    for field in names(table1)
+        if field in discard
+            continue
+        end
+        oe, oc = table1[!,field], table2[!,field]
+
+        if !(≈(oe,oc,rtol=rtol)) 
+            test_result = false
+            break
+        end
+    end
+    @test test_result
+end
+
+@testset "positive_fft_bins" begin
+    freq = fftfreq(11)
+    good = freq .> 0
+    goodbins = positive_fft_bins(11)
+    @test freq[good]==freq[goodbins]
```

Comment:
In Julia masks are less frequently used than in Python, [`filter`](https://docs.julialang.org/en/v1/base/collections/#Base.filter)-ing is much more common:
```suggestion
    goodbins = positive_fft_bins(11)
    @test filter(x -> x >0, freq) == freq[goodbins]
```